comit/src/ethereum.rs

```diff
@@ -227,6 +198,7 @@ pub struct ChainId(u32);
 impl ChainId {
     pub const MAINNET: Self = ChainId(1);
     pub const ROPSTEN: Self = ChainId(3);
```

Comment:
Out of interest, is there a reason we don't have a token contract for Ropsten?

Reply:
Yes, because there is no offical one :(

https://developer.makerdao.com/dai/1/api/

Reply:
There is: `0x31F42841c2db5173425b5223809CF3A38FEde360`
See https://github.com/makerdao/developerguides/blob/804bb1f4d1ea737f0287cbf6480a570b888dd547/dai/dai-token/dai-token.md

Reply:
Nice!

Thank you @D4nte.

But where did you get this address from? For me it says: 0x4F96Fe3b7A6Cf9725f59d353F723c1bDb64CA6Aa

Reply:
NVM, I mixed it up :)